exercises/practice/secret-handshake/.approaches/introduction.md

```diff
@@ -0,0 +1,98 @@
+# Introduction
+
+## Common suggestions
+
+- Perhaps this would be better with bitwise operators? Have you tried that yet?
```

Comment:
This sounds like a comment on some particular code. Maybe this could be rephrased slightly to something like: "Consider using bitwise operators" or something like that

Reply:
Sure

```suggestion
- consider using bitwise operators
```